kolibri/core/settings.py

```diff
@@ -0,0 +1,27 @@
+"""
+Kolibri core settings module
+============================
+
+This module is *not* the project settings but defines variables that can be
+configured through the project settings - and their defaults.
+
+If something in the core uses a custom project setting, it should use a
+naming scheme in django.conf.settings like ``KOLIBRI_*``.
+
+Please avoid depending on settings in kolibri.deployments.default.settings,
+rather define the default here so we don't have multiple layers of defaults.
+"""
+
+from django.conf import settings
+
+# : These plugins are automatically enabled if they exist at the first run of
+# : Kolibri. After that, they are not searched for automatically since that would
+# : require explicitly disabling them in case they were to be removed on purpose
+# : from INSTALLED_APPS.
+WELL_KNOWN_PLUGINS = getattr(
+    settings,
+    'KOLIBRI_WELL_KNOWN_PLUGINS',
+    [
+        'kolibri.plugins.example_plugin',
+    ]
+)
```

Comment:
It'll only be triggered when your `~/.kolibri/.data_version` file changes or if it doesn't exist.

If you are very bothered by problems when switching branches, you can possibly create a hook that sets KOLIBRI_HOME to something suffixed by the branch name? For a lot of my own development, keeping the home when switching branches has been a feature :)

Well-known plugin use cases are like this:

We have "Kolibri". It has some core plugins. Those are enabled by default because they are well-known. This is important because they cannot be hard-coded into INSTALLED_APPS, in that case the user wouldn't be able to disable them.

They are enabled and disabled just like any other plugin, that's in `utils.cli`.

Now, there are also other plugins living outside of Kolibri's namespace and distributed separately. Let's say there's a kolibri-unicef-farsi-book-reader, it's well known to us, so we can add it to the list of well known plugins. That way, if a system is installed with both kolibri and a well-known plugin, upon initializing Kolibri, that plugin will be enabled.